charts/wga/templates/unbound/service.yaml

```
@@ -17,6 +17,9 @@ spec:
   - port: 53
     protocol: UDP
     targetPort: 53
+  - port: 80
+    protocol: TCP
+    targetPort: 80
```

These ports should be named; (`Error: INSTALLATION FAILED: 1 error occurred:
        * Service "wga-unbound" is invalid: [spec.ports[0].name: Required value, spec.ports[1].name: Required value`)

```suggestion
  - port: 53
    protocol: UDP
    targetPort: 53
    name: dns
  - port: 80
    protocol: TCP
    targetPort: 80
    name: welcome
```

---

done